source/extensions/transport_sockets/proxy_protocol/BUILD

```diff
@@ -0,0 +1,26 @@
+load(
+    "//bazel:envoy_build_system.bzl",
+    "envoy_cc_extension",
+    "envoy_package",
+)
+
+licenses(["notice"])  # Apache 2
+
+# Built-in plaintext connection transport socket.
+
+envoy_package()
+
+envoy_cc_extension(
+    name = "proxy_protocol",
```

Comment:
Should this be named `upstream_proxy_protocol` or similar, so it is clear that it is only for use in an upstream context?

Reply:
Yes, very good suggestion